ospfd/ospfd.c

```diff
@@ -1201,7 +1281,13 @@ void ospf_ls_upd_queue_empty(struct ospf_interface *oi)
 void ospf_if_update(struct ospf *ospf, struct interface *ifp)
 {
 	if (!ospf)
```

Comment:
where is this called with ospf == NULL?
If nowhere, please change to `assert(ospf)`

Reply:
remove default call and implemented to return if ospf == NULL